### PR TITLE
Listing page updated with React-Maps and ability to set pins

### PR DESCRIPTION
**Pull request by 08martinm**

### CODE REVIEW

**Comment by natgrigorovskaya** [comment]
:racehorse:

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
 client/public/bundle.js
 node_modules
 *.map
-
+apiKeyStore.js
```
Comment (cassiebaer) [inline .gitignore]: Use env. variables for consistency

**Comment by 08martinm** [comment]
Updated environment variables for react google map

**Comment by cassiebaer** [comment]
👹

**Pull request merged**